lib/ui/pages/workspace_page/components/bottom_bar/bottom_nav_bar.dart

```diff
@@ -42,10 +45,19 @@ class BottomNavBar extends ConsumerWidget {
           ),
           NavigationDestination(
             label: localizations.color,
-            icon: Icon(
-              Icons.check_box_outline_blank,
-              size: 24,
-              color: PaintroidTheme.of(context).onSurfaceColor,
+            icon: InkWell(
+              child: Container(
+                height: 24.0,
+                width: 24.0,
+                decoration: BoxDecoration(
+                  color: currentPaint.color,
+                  border: Border.all(
+                    color: Colors.white,
```

Comment:
We no longer use Colors.___. There is a Paintroid.of(context) you can use. Please do search all files in you code editor where you use any other color definition and convert it to Paintroid.of(context). You can see what colors are available in CustomColors. Pick that name and append Color to the end to access it.

Example: Colors.white -> Paintroid.of(context).onSurfaceColor.

Yes, I am aware the names are stupid, will have to fix that :)

Reply:
If I do this for the files in my `colorpicker` package, that would mean depending on Paintroid project for colors, which is not desirable i guess? So may be I can skip it for the package files and do it only for the project files

Reply:
Yes. Makes sense.